pkg/api/deep_copy_generated.go

```diff
@@ -205,6 +211,24 @@ func DeepCopy_api_Affinity(in Affinity, out *Affinity, c *conversion.Cloner) err
 	} else {
 		out.NodeAffinity = nil
 	}
+	if in.PodAffinity != nil {
```

Comment:
I'm not an api-server expert, but can't you just block copy *annotations?  Do we need deep full type-checking on every copy operation?